<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="code_field">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compile_btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>compile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="run_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_btn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clock_btn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>clock</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="console_field">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>141</width>
      <height>64</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string> console </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="input_field">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="output_field">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>460</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>320</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>input</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>440</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>output</string>
    </property>
   </widget>
   <widget class="QTableView" name="ram_table">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
